### PR TITLE
Added support in save_mesh_obj for triangle meshes

### PR DESCRIPTION
**Pull request by rcvalerio**

### CODE REVIEW

File: windtunnel/pre_processing.py [hunk 1]
```diff
@@ -110,6 +110,13 @@ def compute_object_length(mesh: pv.PolyData):
 
 
 def save_mesh_obj(mesh, dest_object_path):
-    trimesh_mesh = trimesh.Trimesh(vertices=mesh.points,
-                                   faces=mesh.faces.reshape((-1, 4))[:, 1:])
+    faces = mesh.faces
+    if faces.shape[0] % 3 == 0:
```
Comment (hpenedones) [inline windtunnel/pre_processing.py:114]: Please apply the DRY principle here, use the if just o detect the shape_vertices (either 3 or 4) but then the call to Trimesh is the same, passing the variable in the reshape.

File: windtunnel/windtunnel.py [hunk 2]
```diff
@@ -103,6 +103,16 @@ def set_object(self,
         """
 
         mesh = pv.read(object_path)
+
+        if isinstance(mesh, pv.MultiBlock):
```
Comment (hpenedones) [inline windtunnel/windtunnel.py:107]: a bit unclear what this is doing... is it related to the description of this pull request (save triangle meshes) or something else? Perhaps encapsulating this under a function and giving it a clear name would help...

File: windtunnel/display.py [hunk 3]
```diff
@@ -102,8 +103,27 @@ def add_mesh(self,
             opacity=opacity,
             show_edges=show_edges,
             scalars=scalars,
+            clim=clim,
         )
 
+    def add_force_coefficients(self, coefficients: dict):
```
Comment (hpenedones) [inline windtunnel/display.py:109]: Seems a bit unrelated with the title of the Pull request...  Should the title and description be updated (or ideally, add this in separate pull request)?

**Review by hpenedones — CHANGES_REQUESTED** [review]
Thanks @rcvalerio , left some small suggestions

**Comment by rcvalerio** [comment]
[14de76c](https://github.com/inductiva/wind-tunnel/pull/20/commits/14de76c4d23781eb8c6b71cf690e02bd7479877a) was moved to #20

**Comment by rcvalerio** [comment]
[d438616](https://github.com/inductiva/wind-tunnel/pull/20/commits/d4386161d510399aa7df64db330d4e8f2ed2b6e1) was moved to #22

**Comment by rcvalerio** [comment]
[9682dfc](https://github.com/inductiva/wind-tunnel/pull/20/commits/9682dfcc4a8a9c923f7da2e4b1aa98133c42425b) was moved to #23

**Comment by rcvalerio** [comment]
[83b2a0e](https://github.com/inductiva/wind-tunnel/pull/20/commits/83b2a0ebf9beff84c04265c1b2628d9f98ddb339) was moved to #24

**Comment by rcvalerio** [comment]
[33095e3](https://github.com/inductiva/wind-tunnel/pull/20/commits/33095e3bfa04d23b9432fd2c5485196955d4bc76) was moved to #25

**Comment by rcvalerio** [comment]
All the unrelated commits where to moved to isolated pull requests, hence this pull request is no longer needed.

**Pull request closed without merging**